### DISCUSSION TITLE
https://thanhle.blog/vi/blog/so-thich-voi-dam-me-khac-nhau-nhu-nao

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#19] [Ideas]
# https://thanhle.blog/vi/blog/so-thich-voi-dam-me-khac-nhau-nhu-nao

https://thanhle.blog/vi/blog/so-thich-voi-dam-me-khac-nhau-nhu-nao

**Comment by thientran98qb** [comment]
Em hiện tại đang làm Dev nhưng e thực sự chưa có niềm đam mê với nó. Đi làm cứ chán nản mệt mỏi. Nhưng không làm nó thì e ko biết sẽ làm gì. Giờ e đang lạc lõng hoang mang. A có lời khuyên gì không ạ

**Reply by thanhlmm** [reply]
Thú thực là anh cũng không biết lý do em chán nản là gì. Có thể là môi trường đồng nghiệp quá chán, không có connect được; có thể là do công việc quá khó, hoặc quá dễ; hoặc có thể do bản thân có việc riêng khác, không tập trung vào công việc; hoặc có thể là em không thích và không muốn làm công việc này thật.
Nên anh nghĩ nếu em tự trả lời em chán vì lý do gì thì em sẽ có next action tiếp theo. Vấn đề em gặp phải có phải là vấn đề có thể né tránh hay là vấn đề bắt buộc phải vượt qua. Nếu ví dụ em chán vì môi trường thì em nên thử đổi môi trường khác, là vấn đề có thể né tránh. Còn nếu em thấy thực sự chán ngành này, nhưng em vẫn muốn làm tiếp thì đó là vấn đề em phải vượt qua. Đặt mục tiêu tăng lương x2 sau 1 năm cũng là một cách để vượt qua nè em.
